src/js/view/Template.js

```diff
@@ -0,0 +1,21 @@
+import { CARD_STATE } from '../constants/index.js';
+
+const getCarBoardItemTemplate = ({ name, stateList }) => {
+  return `
+    <div class="mr-2">
+      <div class="car-player">${name}</div>
+      ${stateList
+        .filter((state) => state === CARD_STATE.GO)
+        .map((state) => '<div class="forward-icon mt-2">⬇️️</div>')
+        .join('')}
```

Comment:
```suggestion
      ${'<div class="forward-icon mt-2">⬇️️</div>'.repeat(
        stateList.filter((state) => state === CARD_STATE.GO).length
      )}
```
repaet를 통해 작성하면 더 간결하게 보일거같아요~